spec/features/assignments/index_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
```

Comment:
Not in a bad way. I'm impressed with how much you've done in the test suite. Does it all pass?